independent-publisher-connectors/OpenNEM/apiProperties.json

```diff
@@ -0,0 +1,9 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#da3b01",
+    "capabilities": [],
+    "publisher": "Paul Culmsee (sevensigma.com.au)",
+    "stackOwmer": "OpenNEM (https://opennem.org.au/about/)"

```

Comment:
@paulculmsee Can you please remove the website URLs from the `"publisher"` and `"stackOwner"` properties. We have metadata sections in the swagger where you can provide website URLs.